src/client/eppo-client.spec.ts

```diff
@@ -27,16 +26,14 @@ describe('EppoClient E2E test', () => {
     window.localStorage.clear();
     window.sessionStorage.clear();
     mock.setup();
-    mock.get(/randomized_assignment\/config*/, (_req, res) => {
+    mock.get(/randomized_assignment\/v2\/config*/, (_req, res) => {
       const rac = readMockRacResponse();
       return res.status(200).body(JSON.stringify(rac));
     });
     const preloadedConfig = {
       name: preloadedConfigExperiment,
-      percentExposure: 1,
       enabled: true,
       subjectShards: 100,
-      variations: mockVariations,
```

Comment:
`variations` and `percentExposure` no longer live at the top level, they live on individual allocations.